src/crypto/tls/conn.go

```diff
@@ -50,6 +50,9 @@ type Conn struct {
 	// verifiedChains contains the certificate chains that we built, as
 	// opposed to the ones presented by the server.
 	verifiedChains [][]*x509.Certificate
+	// verifiedDC is set by the peer who negotiates the use of a valid delegated
+	// credential.
```

Comment:
"is set" -> "is provided by" or "contains", to clarify that the purpose of this field?
```
verifiedDC contains a delegated credential from the peer and
has been verified against the leaf certificate.
```

Reply:
Yes! Thank you!